src/validators/orderformValidator.js

```diff
@@ -0,0 +1,13 @@
+export function validateHour (hourInput) {
+    const pattern = /([0-9]{1,2})\:([0-9]{2})/;
```

Comment:
add ^ and $ to the start and end of the regex, cause this passes for `asdf4:0asdf`